README.rst

```diff
@@ -0,0 +1,134 @@
+	python-matplotlib
+
+To install the remaining packages, see installation instructions at their
+respective web-pages.
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
+run. If you are unfamiliar with the CLI, consult online resources on how to get 
+started with it on your machine.
+
+If you're using Anaconda Python, you can install from the "conda-forge" 
+channel of Anaconda packages ::
+
+     conda config --add channels conda-forge
+     conda install diffpy.pdfmorph
+
+If you don't use Anaconda or prefer to install from sources, make sure the required
+software is all in place and run ::
+	
+	python setup.py install
+
+By default the files are installed to standard system directories, which may require
+the use of ``sudo`` for write privileges. If administrator (root) access is not
+available, see the output from ``python setup.py install --help`` for options to 
+install as a regular user to user-writable locations. Note that installation to
+non-standard directories may require adjustments to the PATH and PYTHONPATH 
+environment variables.
+
+To ensure the installation worked, activate the relevant conda environment and 
+type ::
+
+	pdfmorph --version
+
+If installed correctly, this command should return PDFmorph's current version
+number.
+
+PDFmorph can then be started from a terminal ("Anaconda Prompt" on Windows) by
+executing the ``pdfmorph`` program.
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+# FIXME - add reference to online instrucitons once merged
+For more information on how to use PDFmorph, please consult online documentation
+[here].
+
+
+DEVELOPMENT
+------------------------------------------------------------------------
+
+PDFmorph is an open-source software available in a git repository at
+https://github.com/diffpy/diffpy.pdfmorph.
+
+Feel free to fork the project and contribute! To install PDFmorph
+in a development mode where the source files are used directly
+rather than copied to a system directory, use ::
+
+     python setup.py develop --user
+
+
+CONTACTS
+------------------------------------------------------------------------
+
+# FIXME - add link to diffpy.org/products/pdfmorph.html when that exists
+For more information on PDFmorph, visit the `PDFmorph project web-page. <https://github.com/diffpy/diffpy.github.io/blob/source/products/pdfmorph.rst/>`_
+
+or email Professor Simon Billinge at sb2896@columbia.edu
+
+
+LICENSE
+------------------------------------------------------------------------
+
+For full license and copyright information, check `here. <https://github.com/diffpy/diffpy.pdfmorph/blob/master/LICENSE.txt/>`_ 
```

Comment:
This section should be near the top of the readme, right before REQUIREMENTS A statement something like "this software is subject to license and copyright restrictions listed `here`

Reply:
done!